src/middleware/transactionBody.js

```diff
@@ -0,0 +1,26 @@
+import mongodb from 'mongodb'
+
+export const retrieveTransactionBody = (db, fileId, callback) => {
```

Comment:
I dont think its nessecary to pass throught the db connection to the function. A connection already exists which we can re-use
https://github.com/jembi/openhim-core-js/pull/1019/files#diff-6b387023209fd79314641ba533f5ce23R3
https://github.com/jembi/openhim-core-js/pull/1019/files#diff-6b387023209fd79314641ba533f5ce23R48